### DISCUSSION TITLE
renovate.json config for skipping a whole repository

### DISCUSSION THREAD

**Discussion by kad-meedel** [renovatebot/renovate#25188] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

Gitea or Forgejo

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

A possibility to skip a whole repository. For Instance a document repository which doesn't need to be checked.
So I need a renovate.json config within that specific repository for skipping the scan.

### What you tried so far.

Searching  the documentation site: https://docs.renovatebot.com/configuration-options/  and the world wide web for possible solutions.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
"enabled": false

**Reply by kad-meedel** [reply]
Hello Rarkins, I wil look into it. Thanks for your reply
